src/components/BatchSettings.vue

```diff
@@ -18,7 +18,7 @@
             <div>{{ $t('Base.downloadTemplatePrompt') }}</div>
             <ul>
               <li>{{ $t('Base.batchSettingDownloadFirst') }}</li>
-              <li>{{ $t('Base.moreImportInstructions') }}</li>
+              <li v-html="$t('Base.moreImportInstructions', { link: checkDocMap[type] })"></li>
```

Comment:
can we use https://vue-i18n.intlify.dev/guide/advanced/component.html#basic-usage ?

Reply:
Sure, this is because I am too lazy to use it. 😅

Reply:
Updated